util/users.py

```diff
@@ -55,3 +55,21 @@ async def delete_user(identifier: user_models.UserIdentifier) -> None:
     if response.deleted_count == 0:
         detail = "User not found and could not be deleted"
         raise exceptions.UserNotFoundException(detail=detail)
+
+
+async def validate_user_info(
+        validate: user_models.ValidateUserForm) -> str:
+    """
+    Validates user based off identifier and password
+    #todo: call get_user_info_by_identifier w identifier, validate pass vs user object. return 200
+    """
+    try:
+        user_info = await get_user_info_by_identifier(validate.identifier)
+    except exceptions.UserNotFoundException:
+        return 'not found'
```

Comment:
If the goal of this try/except block is to account for non-existent users, then it is actually redundant.

If you see the code in `get_user_info_by_identifier`, it will raise it's own `exception` (the same one we're catching here). So this is effectively catching it to re-raise it.

Consider instead just `user_info = await get_user_info_by_identifier(validate.identifier)`, which will raise a 404 by itself anyway.

NOTE: this isn't totally clear in the `get_user_info_by_identifier` docstring, but it should be. Bonus points if you add a line or two to that method's documentation that details the fact that it will raise a 404 `UserNotFoundException` if no user is found (boy scout rule).